src/query/query.pest

```diff
@@ -0,0 +1,24 @@
+query         =  { SOI ~ interfaceExpr ~ EOI }
+interfaceExpr = _{ expr | string }
+// Interface expr can a standard expression or literal ID
+expr = _{ singleVariableFunc | noVariableFunc | infixExpression }
+
+noVariableFunc     =  { funcNames ~ "()" | latest }
```

Comment:
Do some basic prep beforehand to process literal ids into `single(id == 'literal')` and `latest` into `latest()`, then simplify the grammar

Reply:
Thanks, have tidied this up